minibear/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -0,0 +1,545 @@
+    // basic x translation negative one meter
+    this.driverController.a().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(-1.0, 0.0, 0.0)),
+            new ButtonPressCommand(
+                "driverController.a()",
+                "-1.0m X translation DriveToPointCommand"))
+            .withTimeout(10.0));
+    // basic x translation positive one meter
+    this.driverController.y().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(1.0, 0.0, 0.0)),
+            new ButtonPressCommand(
+                "driverController.y()",
+                "1.0m X translation DriveToPointCommand"))
+            .withTimeout(10.0));
+    // basic y translation negative one meter
+    this.driverController.b().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(0.0, -1.0, 0.0)),
+            new ButtonPressCommand(
+                "driverController.b()",
+                "-1.0m y translation DriveToPointCommand"))
+            .withTimeout(10.0));
+    // basic y translation positive one meter
+    this.driverController.x().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(0.0, 1.0, 0.0)),
+            new ButtonPressCommand(
+                "driverController.x()",
+                "1.0m y translation DriveToPointCommand"))
+            .withTimeout(10.0));
+    // basic rotation of 90 degrees
+    this.driverController.leftBumper().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(0.0, 0.0, 90.0)),
+            new ButtonPressCommand(
+                "driverController.leftBumper()",
+                "90 degree rotation DriveToPointCommand"))
+            .withTimeout(10.0));
+    // basic rotation of -90 degrees
+    this.driverController.rightBumper().onTrue(
+        new ParallelCommandGroup(
+            new DriveToPointCommand(
+                this.subsystemCollection.getDriveTrainSubsystem(),
+                this.getTargetPosition(0.0, 0.0, -90.0)),
+            new ButtonPressCommand(
+                "driverController.rightBumper()",
+                "-90 degree rotation DriveToPointCommand"))
+            .withTimeout(10.0));
+  }
+
+  /**
+   * A method that will bind buttons to have the robot flow through various
+   * trajectories
+   */
+  private void bindDriveTrajectoryButtonsToDriverXboxController() {
```

Comment:
remove this. unused. Don't need to make them controller buttons, for testing, we can add buttons to the shuffleboard.

Reply:
done.